copier/main.py

```diff
@@ -290,11 +304,19 @@ def render_folder(dst_path, rel_folder, flags):
         printf("create", display_path, style=STYLE_OK)
 
 
-def render_file(dst_path, rel_path, source_path, render, must_skip, flags):
+def render_file(
+    dst_path: Path,
+    rel_path: Path,
+    source_path: Path,
+    engine: Renderer,
+    must_skip: CheckPathFunc,
+    flags: Dict[str, bool],
+) -> None:
     """Process or copy a file of the skeleton.
     """
+    content: Optional[str]
     if source_path.suffix == ".tmpl":
-        content = render(source_path)
+        content = engine(source_path)
```

Comment:
I think `render` was a much better name. Without knowing what `engine` is, `engine(something)` is a mistery. I'm going to change it back.